wdl-doc/src/callable/task.rs

```diff
@@ -26,82 +32,75 @@ pub struct Task {
     outputs: Vec<Parameter>,
     /// The runtime section of the task.
     runtime_section: Option<RuntimeSection>,
+    /// The command section of the task.
+    command_section: Option<CommandSection>,
 }
 
 impl Task {
     /// Create a new task.
-    pub fn new(
-        name: String,
-        meta_section: Option<MetadataSection>,
-        parameter_meta: Option<ParameterMetadataSection>,
-        input_section: Option<InputSection>,
-        output_section: Option<OutputSection>,
-        runtime_section: Option<RuntimeSection>,
-    ) -> Self {
-        let meta = match meta_section {
+    pub fn new(name: String, version: SupportedVersion, defintion: TaskDefinition) -> Self {
```

Comment:
```suggestion
    pub fn new(name: String, version: SupportedVersion, definition: TaskDefinition) -> Self {
```

Reply:
Probably once, plus 6 autocompletes. TBF, it wouldn't have compiled if it were only misspelled once.